src/Workspaces/CoreTest/SolutionTests/SolutionWithSourceGeneratorTests.cs

```diff
@@ -733,7 +732,7 @@ public async Task ForkAfterFreezeNoLongerRunsGenerators()
         [Fact, WorkItem(56702, "https://github.com/dotnet/roslyn/issues/56702")]
         public async Task ForkAfterFreezeNoLongerRunsGeneratorsEvenIfCompilationFallsAwayBeforeFreeze()
```

Comment:
This test was mooted once we went to strong compilation holding, and can just be deleted.

Reply:
mooted.